design/sketch.webidl

```diff
@@ -169,12 +169,12 @@ interface GPUTextureUsage {
 
 dictionary GPUTextureDescriptor {
     GPUExtent3D size;
```

Comment:
I think we actually need `required` here and on most other fields that don't have defaults.